css/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+	box-sizing: border-box;
+	font-family: "Times New Roman", Times, serif;
+}
+
+h1 {
```

Comment:
Do you always want to apply this to all h1 elements? Consider using a class instead.